proposals/reward-proposal-v2-reward-distribution.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Epoch Reward Distribution
+---
+
+## Problem
+
+In the `Epoch Reward Calculation V2`, we proposed a new method to handle
+calculating epoch rewards. This is the second part of that proposal - `Reward
+Distribution`.
+
+## Proposed Solutions
+
+Similar to [Partitioned Inflationary Rewards
+Distribution](https://github.com/solana-labs/solana/blob/master/docs/src/proposals/partitioned-inflationary-rewards-distribution.md),
+the rewards are distributed over `M` slots.
```

Comment:
* What is `M` ? and why do we need?
* Can't we distribute rewards just as soon as we can regardless of a fixed value `M`?
* If we need a fixed distribution, why pick a fixed `M` slots instead of a fixed number of accounts per slot?